DigitalOffice/Services/ProjectService/Validators/EditProjectValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class EditProjectValidator : AbstractValidator<EditProjectRequest>
+    {
+        public EditProjectValidator()
+        {
+            RuleFor(project => project.Id)
+                .Must(NotGuidEmpty)
+                .WithMessage("Request must have a project Id");
+
+            RuleFor(project => project.Name)
+                    .NotEmpty()
+                    .WithMessage("Project must have a name.")
+                    .MaximumLength(80)
+                    .WithMessage("Project name too long.")
+                    .Matches("^[A-Z 0-9][A-Z a-z 0-9]+$")
+                    .WithMessage("Incorrect project name.");
+
+            RuleFor(project => project.Description)
+                .MaximumLength(500)
+                .WithMessage("Project description is too long.");
+        }
+
+        private bool NotGuidEmpty(Guid guid)
+        {
+            if (guid == Guid.Empty || guid == null)
+            {
+                return false;
+            }
+            return true;
+        }
+    }
```

Comment:
Why not use a simple check .NotEmpty()? Guid can not be null

Reply:
It did not work properly when "0000000" guid was passed. So this method will remain.

Reply:
I'll just remove "== null" check

Reply:
Why? I wrote test for validator and validator had validation exception when Guid is empty

Reply:
jesus you wont let me close this pr will ya

Alright, I think this was added because it was nullable so there were conflicts. Definetily.

And also that is why Guid was checked to be null because it was written as GUID?

Anyway I checked and it's working. Will fix